test/jdk/javax/swing/JTableHeader/6889007/bug6889007.java

```diff
@@ -83,7 +84,7 @@ public static void main(String[] args) throws Exception {
             int y = point.y + height/2;
             for(int i = -shift; i < width + 2*shift; i++) {
                 robot.mouseMove(x++, y);
-                robot.waitForIdle();
+                robot.delay(100);
```

Comment:
It looks like we don't need this delay since there is `robot.setAutoDelay(100);` on line 56. A total delay of 200ms seems too much to me.

Reply:
I dont think it's too much as compared to 500ms we have in some tests..Since I have tested in all platforms with this values without any problems observed, I will like to keep this value for now..

Reply:
This seems OK to me for a rarely repeated operation.
Here it is repeated 405 times(at least for my testing), which leads to an additional ~40.5 s of the test execution time.
100ms is still a pretty solid delay for mouse movement.

Reply:
ok

Reply:
Removed setAutoDelay as there are no other robot usage apart from the mouseMove where I retained the delay..
Testing is green.